pkg/chartmuseum/logging.go

```diff
@@ -0,0 +1,107 @@
+package chartmuseum
```

Comment:
Should'nt we also add a logging package here to further separation ?

Reply:
Do you mean move this out of pkg/chartmuseum? I like the idea of keeping all server-related stuff in here, new file simply to make server.go less cluttered

Reply:
I meant:
```golang
package logging
```

Reply:
Oh, I'm not sure that makes sense. If we were to do that, we would probably want to break it out into its own directory (pkg/<package_name>/logger.go).

For example, see package declarations in all files here, they all have `package kube`: https://github.com/kubernetes/helm/tree/master/pkg/kube

Reply:
nevermind